File: JavaScript/fronts.js
class Node {
    constructor(data) {
        this.data = data;
        this.next = null;
    }
}

class linkedList {
    constructor() {
        this.head = null;
        this.size = 0;
    }

    front() { //return the value at the head of the list 
        if(this.head != null)
            return this.head.data;
    }

    removeFront() { // remove head value from front of list
        if(this.head != null)
            this.head = curr.next;
    }

    add(data) { //add an element to the list 
        var node = new Node(data);
        var current;

        if (this.head == null)
        this.head = node;
        else {
            current = this.head

            while (current.next) {
                current = current.next;
            }
            current.next = node;
        }
        this.size++;
    }
}

File: Python/fronts.py
class Node:
    def __init__(self, val):
        self.val = val
        self.next = None 

class List:
    def __init__(self):
        self.head = None
    
    #Front value 
    def front(self):
        if self.head:
            return self.head.val
        
    #Remove front 
    def remove_front(self):
        if self.head:
            self.head = self.head.next

    #Add to the front 
    def add_front(self,val):
        new_node = Node(val)
        if not self.head:
            self.head = new_node
        else:
            new_node.next = self.head
            self.head = new_node